notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -117,7 +117,8 @@
    "metadata": {},
```

Comment:
<p>Very nice!</p><p> </p><p>With the object oriented proposal, this would be: <code>plot_results(exp_list, "test_score")</code></p><p> </p><p>Maybe we can remove <code>stats_list</code> as parameter for now, since you don't use it + I don't understand which are the "two variables" that you are referring to in the docstring? It would also leave the question of statistical significance to the next PR.</p><p> </p><p>If we go the route of comparing the effects of statistics versus geometry, we could start organizing the experiments in:</p><ul><li>2D bar plots, where each line represents a geometric method, and each column a statistical method</li><li>Table, where each line represents a geometric method, and each column a statistical method and we report mean, std dev in each cell.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>